src/Util/TestDox/TestResult.php

```diff
@@ -9,147 +9,19 @@
  */
 namespace PHPUnit\Util\TestDox;
 
-final class TestResult
```

Comment:
btw, this is a BC breaker for all of us that were using this class. (class is not marked as `@internal`)
making it non-class (->interface) is even bigger change